src/pages/Welcome.scss

```diff
@@ -0,0 +1,48 @@
+@import "../styles/Colors.scss";
+@import "../styles/Mixins.scss";
+@import "../styles/Fonts.scss";
+.welcome-page {
+    height: 100vh;
+    @include centerFlex;
+    font-size: $smallFont;
+    color: $darkBlue;
+    background-color: $lightBlue;
+    background: linear-gradient(0deg, rgba(40, 103, 178, 0.28895308123249297) 0%, rgba(255, 255, 255, 1) 71%);
```

Comment:
gradient also should be added to colors because it will be main theme for other pages - it should be name like more less like this